RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
Copy-pasted old implementation just can not stay.

Your PR changes the code in a way that this old implementation is not called in the regular ECAL local reco workflows.
The interface still remains for a possibility of direct calls given by digi.
I'm trying to suggest alternatives: one of them comes at nearly zero extra cost and preserves functionality, which is probably a dead end anyways (that's why an exception is another valid solution). Possibly there are other options, but I haven't come up with any.

Thank you.